.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+# https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/ci-configurations/github-actions.md
+
+name: Release
+
+on:
+  workflow_dispatch:
```

Comment:
I have no issues with running this manually vs. using the release workflow but I am curious why to go this route vs. the release trigger:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Is this so we can use the automatic semver without issue? Either way, looks good.

Reply:
Because it creates the release automatically, so this way you just click an action and it will automatically determine the correct version and notes to put in the release. I think that's better than manually creating a release where it's possible to put the wrong version etc

Reply:
Perfect, makes sense to me. As long as we can still create GitHub release notes should be fine. At this point we can probably cleanup the readme and remove some of the old release notes or move them to the GitHub releases section anyway.